source/integration_tools.html.md

```diff
@@ -0,0 +1,73 @@
+
+# Credentials
+
+Credentials issued via Integration Tools grants Apps and Integrations access to all the data in the Welkin environment. These `client_id` and `client_secret` pair generated by Integration Tools is for syncing data for your entire Welkin environment and is not scoped to specific patients or workers within your Environment.
+
+Each App or Integration you configure for your Welkin Environment should have its own set of credentials for accessing data. Granting separate credentials for each system which has access to Welkin data ensures that if you want to disable a single system you can do this without effecting others.
+
+Credentials created in Integration Tools are active as soon as they are created and can be used to generate access tokens. Access tokens are short lived and are used to make calls to the API. Details on this token exchange process is [documented here](/data_api.html/#authentication).
+
+<aside>When you invalidate Credentials in Integration Tools, Access Tokens are NOT invalidated. They will expire as normal. The credentials will no longer be valid for generating new Access Tokens.</aside>
+
+# Webhooks
+
+Webhooks specified via Integration Tools send notifications to Apps and Integrations when data is updated in Welkin.
+
+In Integration Tools you select the destination for these webhooks, the method used to secure the data transfer of webhooks, and the credentials used in the selected security method.
+
+When Welkin adds new API endpoints we will not automatically opt in notifications to be sent for those new data types. You must explicitly opt in for each type of notification you want to receive.
+
+You can pause webhooks from this page which prevents us from sending webhooks to your system.
+
+# Environments vs Configurations
+
+Environments in Welkin are the entities the house and separate patient data. Each Environment has a separate set of patients and workers.
+
+Configurations in Welkin are the collection of settings which define how Welkin will work.
+
+Configurations are published to Environments and control how that Environment will operate.
+
+Integrations and Apps are attached to specific Environments. Credentials give these Apps access to only the data housed within that Environment.
+
+The data housed within an Environment is controlled by the schema defined in the Configuration which is applied to that Environment.
+
+Integration Tools edits Environments and not Configurations.
+
+# Limitations
+
+## One webhook per environment
+
+Currently you can only configure one webhook per environment. This limits how many services can receive data from Welkin via webhook.
+
+## Webhooks must used valid credential
+
+You can only select valid credentials for securing Webhooks. If you try to invalidate credentials you must first remove them from use with Webhooks.
```

Comment:
> If you try to invalidate credentials you must first remove them from use with Webhooks

Do we prevent doing that? Or this a "you really should remove them from use with Webhooks"?

Reply:
we prevent you